### DISCUSSION TITLE
❓ Questions  Have a doubt about samples

### DISCUSSION THREAD

**Discussion by moodeerf** [snakers4/silero-vad#219] [Q&A] [answered]
## ❓ Questions and Help
I don't understand that the samples of 30ms is 512 rather than 480 when the sample rate is 16000, Could you show more details about how to calculate it ? Thank you very much.
We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.

**Comment by snakers4** [comment, marked as answer]
Hi,

This is answered here 

https://github.com/snakers4/silero-vad/discussions/211
